src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,9 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private static final UserService userService = new UserService();
```

Comment:
services are not constants usually
```suggestion
    private final UserService userService = new UserService();
```